# English translations for vigilo-correlator.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the vigilo-correlator
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2015.
#
msgid ""
msgstr ""
"Project-Id-Version: vigilo-correlator 3.5\n"
"Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
"POT-Creation-Date: 2015-04-22 09:58+0200\n"
"PO-Revision-Date: 2015-04-22 09:58+0200\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: en <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/correlator/__init__.py:33
msgid "The topology has been reloaded."
msgstr "The topology has been reloaded."

#: src/vigilo/correlator/__init__.py:53
msgid ""
"Could not set signal handlers. The correlator may not be able to shutdown"
" cleanly"
msgstr ""
"Could not set signal handlers. The correlator may not be able to shutdown"
" cleanly"

#: src/vigilo/correlator/correvent.py:122
#, python-format
msgid ""
"Ignoring request to update correlated event %r: a more recent update "
"already exists in the database"
msgstr ""
"Ignoring request to update correlated event %r: a more recent update "
"already exists in the database"

#: src/vigilo/correlator/correvent.py:127
#, python-format
msgid "Updating existing correlated event (%r)"
msgstr "Updating existing correlated event (%r)"

#: src/vigilo/correlator/correvent.py:130
#, python-format
msgid "Got a reference to a non-existent correlated event (%r), adding as new"
msgstr "Got a reference to a non-existent correlated event (%r), adding as new"

#: src/vigilo/correlator/correvent.py:173
#, python-format
msgid ""
"Got a reference to a nonexistent correlated event (%r), skipping this "
"aggregate"
msgstr ""
"Got a reference to a nonexistent correlated event (%r), skipping this "
"aggregate"

#: src/vigilo/correlator/correvent.py:363
#, python-format
msgid ""
"Creating new aggregate with cause #%(cause)d (#%(supitem)d) from "
"aggregate #%(original)d"
msgstr ""
"Creating new aggregate with cause #%(cause)d (#%(supitem)d) from "
"aggregate #%(original)d"

#: src/vigilo/correlator/correvent.py:472
msgid "Sending the correlated event to syslog"
msgstr "Sending the correlated event to syslog"

#: src/vigilo/correlator/correvent.py:560
#, python-format
msgid ""
"Ignoring event #%(idevent)d on (%(host)r, %(service)r) as requested by "
"the correlation rules"
msgstr ""
"Ignoring event #%(idevent)d on (%(host)r, %(service)r) as requested by "
"the correlation rules"

#: src/vigilo/correlator/correvent.py:586
#, python-format
msgid "Raw event ignored. Reason: status = %r"
msgstr "Raw event ignored. Reason: status = %r"

#: src/vigilo/correlator/correvent.py:603
msgid "Creating a new correlated event"
msgstr "Creating a new correlated event"

#: src/vigilo/correlator/db_insertion.py:54
#, python-format
msgid "Received request to add an event on HLS \"%s\""
msgstr "Received request to add an event on HLS \"%s\""

#: src/vigilo/correlator/db_insertion.py:59
#, python-format
msgid ""
"Got a reference to a non configured item (%(host)r, %(service)r), "
"skipping event"
msgstr ""
"Got a reference to a non configured item (%(host)r, %(service)r), "
"skipping event"

#: src/vigilo/correlator/db_insertion.py:73
#, python-format
msgid ""
"Got a notification about an UNKNOWN state for service \"%(service)s\" on "
"unreachable host \"%(host)s\", skipping event"
msgstr ""
"Got a notification about an UNKNOWN state for service \"%(service)s\" on "
"unreachable host \"%(host)s\", skipping event"

#: src/vigilo/correlator/db_insertion.py:125
#, python-format
msgid ""
"Ignoring request to create a new event with state \"%s\" (nothing "
"alarming here)"
msgstr ""
"Ignoring request to create a new event with state \"%s\" (nothing "
"alarming here)"

#: src/vigilo/correlator/db_insertion.py:132
msgid "Creating new event"
msgstr "Creating new event"

#: src/vigilo/correlator/db_insertion.py:137
msgid "Multiple raw events found, using the first one available."
msgstr "Multiple raw events found, using the first one available."

#: src/vigilo/correlator/db_insertion.py:142
#, python-format
msgid "Updating event %r"
msgstr "Updating event %r"

#: src/vigilo/correlator/db_insertion.py:201
#, python-format
msgid "Got a reference to a non configured high-level service (%(service)r)"
msgstr "Got a reference to a non configured high-level service (%(service)r)"

#: src/vigilo/correlator/db_insertion.py:228
#, python-format
msgid ""
"Got a reference to a non configured item (%(host)r, %(service)r), "
"skipping state"
msgstr ""
"Got a reference to a non configured item (%(host)r, %(service)r), "
"skipping state"

#: src/vigilo/correlator/db_insertion.py:304
#, python-format
msgid "Adding event #%(event)d (supitem #%(supitem)d) to aggregate #%(aggregate)d"
msgstr "Adding event #%(event)d (supitem #%(supitem)d) to aggregate #%(aggregate)d"

#: src/vigilo/correlator/db_insertion.py:320
#, python-format
msgid ""
"Event #%(event)d already belongs to aggregate #%(aggregate)d, refusing to"
" add it twice"
msgstr ""
"Event #%(event)d already belongs to aggregate #%(aggregate)d, refusing to"
" add it twice"

#: src/vigilo/correlator/db_insertion.py:371
#, python-format
msgid "Merging aggregate #%(src)d into aggregate #%(dest)d"
msgstr "Merging aggregate #%(src)d into aggregate #%(dest)d"

#: src/vigilo/correlator/db_insertion.py:442
#: src/vigilo/correlator/db_insertion.py:447
msgid "Got a reference to a nonexistent aggregate, aborting"
msgstr "Got a reference to a nonexistent aggregate, aborting"

#: src/vigilo/correlator/db_insertion.py:459
#, python-format
msgid ""
"Event #%(event)d (supitem #%(supitem)d) will be merged into aggregate "
"#%(aggregate)d"
msgstr ""
"Event #%(event)d (supitem #%(supitem)d) will be merged into aggregate "
"#%(aggregate)d"

#: src/vigilo/correlator/handle_ticket.py:35
#, python-format
msgid ""
"handle_ticket: Trouble ticket message received. Timestamp = "
"%(timestamp)r. Impacted HLS = %(hls)r. Ticket id = %(ticket_id)r. "
"acknowledgement_status = %(ack_status)r. Message = %(message)r."
msgstr ""
"handle_ticket: Trouble ticket message received. Timestamp = "
"%(timestamp)r. Impacted HLS = %(hls)r. Ticket id = %(ticket_id)r. "
"acknowledgement_status = %(ack_status)r. Message = %(message)r."

#: src/vigilo/correlator/handle_ticket.py:57
#, python-format
msgid "handle_ticket: No matching trouble ticket found : %r"
msgstr "handle_ticket: No matching trouble ticket found : %r"

#: src/vigilo/correlator/handle_ticket.py:63
#, python-format
msgid "handle_ticket: Several events seem to be associated with this ticket : %r"
msgstr "handle_ticket: Several events seem to be associated with this ticket : %r"

#: src/vigilo/correlator/handle_ticket.py:66
#, python-format
msgid ""
"handle_ticket: The event %(event_id)r is associated with the given ticket"
" (%(ticket_id)r)"
msgstr ""
"handle_ticket: The event %(event_id)r is associated with the given ticket"
" (%(ticket_id)r)"

#: src/vigilo/correlator/handle_ticket.py:88
#, python-format
msgid "handle_ticket: Got exception while updating event %r history"
msgstr "handle_ticket: Got exception while updating event %r history"

#: src/vigilo/correlator/handle_ticket.py:92
#, python-format
msgid "handle_ticket: Event %r history updated successfully."
msgstr "handle_ticket: Event %r history updated successfully."

#: src/vigilo/correlator/memcached_connection.py:65
#, python-format
msgid "Connected to memcached (%s)"
msgstr "Connected to memcached (%s)"

#: src/vigilo/correlator/memcached_connection.py:82
#, python-format
msgid "Connecting to memcached server (%s)..."
msgstr "Connecting to memcached server (%s)..."

#: src/vigilo/correlator/memcached_connection.py:91
#, python-format
msgid "Connection to memcached (%(conn)s) lost: %(reason)s"
msgstr "Connection to memcached (%(conn)s) lost: %(reason)s"

#: src/vigilo/correlator/memcached_connection.py:104
#, python-format
msgid "Connection to memcached (%(conn)s) failed: %(reason)s"
msgstr "Connection to memcached (%(conn)s) failed: %(reason)s"

#: src/vigilo/correlator/memcached_connection.py:242
#, python-format
msgid "Trying to set value '%(value)s' for key '%(key)s' (transaction=%(txn)r)."
msgstr "Trying to set value '%(value)s' for key '%(key)s' (transaction=%(txn)r)."

#: src/vigilo/correlator/memcached_connection.py:285
#, python-format
msgid "Trying to get the value of the key '%(key)s' (transaction=%(txn)r)."
msgstr "Trying to get the value of the key '%(key)s' (transaction=%(txn)r)."

#: src/vigilo/correlator/memcached_connection.py:318
#, python-format
msgid "Trying to delete the key '%(key)s' (transaction=%(txn)r)."
msgstr "Trying to delete the key '%(key)s' (transaction=%(txn)r)."

#: src/vigilo/correlator/registry.py:59
#, python-format
msgid ""
"Rule %(name)s (%(class)s) has already been registered, ignoring attempt "
"to re-register it."
msgstr ""
"Rule %(name)s (%(class)s) has already been registered, ignoring attempt "
"to re-register it."

#: src/vigilo/correlator/registry.py:70
#, python-format
msgid "Cyclic dependencies found: %(path)s"
msgstr "Cyclic dependencies found: %(path)s"

#: src/vigilo/correlator/registry.py:77
#, python-format
msgid "Successfully registered rule %r"
msgstr "Successfully registered rule %r"

#: src/vigilo/correlator/registry.py:142
msgid ""
"Singleton has already been instanciated, ignoring attempt to create a new"
" instance."
msgstr ""
"Singleton has already been instanciated, ignoring attempt to create a new"
" instance."

#: src/vigilo/correlator/registry.py:186
#, python-format
msgid "Not a valid rule name \"%s\""
msgstr "Not a valid rule name \"%s\""

#: src/vigilo/correlator/registry.py:192
#, python-format
msgid "Unable to load rule \"%(name)s\" (%(class)s)"
msgstr "Unable to load rule \"%(name)s\" (%(class)s)"

#: src/vigilo/correlator/registry.py:210
#, python-format
msgid ""
"The rule \"%(rule)s\" depends on the rule \"%(dependency)s\", which has "
"not been loaded."
msgstr ""
"The rule \"%(rule)s\" depends on the rule \"%(dependency)s\", which has "
"not been loaded."

#: src/vigilo/correlator/actors/executor.py:87
#, python-format
msgid "Rule %(rule_name)s timed out"
msgstr "Rule %(rule_name)s timed out"

#: src/vigilo/correlator/actors/rule_dispatcher.py:94
#, python-format
msgid "Unable to contact the database: %s"
msgstr "Unable to contact the database: %s"

#: src/vigilo/correlator/actors/rule_dispatcher.py:132
msgid "Received invalid item ID (None)"
msgstr "Received invalid item ID (None)"

#: src/vigilo/correlator/actors/rule_dispatcher.py:145
#, python-format
msgid "Unexpected error: %s"
msgstr "Unexpected error: %s"

#: src/vigilo/correlator/actors/rule_dispatcher.py:168
msgid "Error while modifying the ticket"
msgstr "Error while modifying the ticket"

#: src/vigilo/correlator/actors/rule_dispatcher.py:180
msgid "Error while retrieving supervised item ID"
msgstr "Error while retrieving supervised item ID"

#: src/vigilo/correlator/actors/rule_dispatcher.py:193
msgid ""
"The rule "
"'vigilo.correlator_enterprise.rules.hls_deps:HighLevelServiceDepsRule' "
"must be loaded for computation orders to be handled properly."
msgstr ""
"The rule "
"'vigilo.correlator_enterprise.rules.hls_deps:HighLevelServiceDepsRule' "
"must be loaded for computation orders to be handled properly."

#: src/vigilo/correlator/actors/rule_dispatcher.py:203
#: src/vigilo/correlator/actors/rule_dispatcher.py:289
msgid ""
"The connection to memcached timed out. The message will be handled once "
"more."
msgstr ""
"The connection to memcached timed out. The message will be handled once "
"more."

#: src/vigilo/correlator/actors/rule_dispatcher.py:310
msgid "Inserting state"
msgstr "Inserting state"

#: src/vigilo/correlator/actors/rule_dispatcher.py:312
msgid "Error while saving state"
msgstr "Error while saving state"

#: src/vigilo/correlator/actors/rule_dispatcher.py:332
msgid "Inserting an entry in the HLS history"
msgstr "Inserting an entry in the HLS history"

#: src/vigilo/correlator/actors/rule_dispatcher.py:334
msgid "Error while adding an entry in the HLS history"
msgstr "Error while adding an entry in the HLS history"

#: src/vigilo/correlator/actors/rule_dispatcher.py:339
msgid "Inserting an entry in the history"
msgstr "Inserting an entry in the history"

#: src/vigilo/correlator/actors/rule_dispatcher.py:341
msgid "Error while adding an entry in the history"
msgstr "Error while adding an entry in the history"

#: src/vigilo/correlator/actors/rule_dispatcher.py:355
msgid "Actual correlation"
msgstr "Actual correlation"

#: src/vigilo/correlator/actors/rule_dispatcher.py:389
#, python-format
msgid "Correlation process ended (%.4fs)"
msgstr "Correlation process ended (%.4fs)"

#: src/vigilo/correlator/actors/rule_dispatcher.py:407
msgid "Handling correlation results"
msgstr "Handling correlation results"

#: src/vigilo/correlator/actors/rule_dispatcher.py:426
#, python-format
msgid ""
"Error while saving the correlated event (%s). The message will be handled"
" once more."
msgstr ""
"Error while saving the correlated event (%s). The message will be handled"
" once more."

#: src/vigilo/correlator/actors/rule_dispatcher.py:455
#, python-format
msgid "Correlation failed for message #%(id)s (%(payload)s)"
msgstr "Correlation failed for message #%(id)s (%(payload)s)"

#: src/vigilo/correlator/actors/rule_dispatcher.py:475
#, python-format
msgid ""
"Unable to store correlated alert for message #%(id)s (%(payload)s) : "
"%(error)s"
msgstr ""
"Unable to store correlated alert for message #%(id)s (%(payload)s) : "
"%(error)s"

#: src/vigilo/correlator/actors/rule_dispatcher.py:508
#, python-format
msgid "%s. The message will be handled once more."
msgstr "%s. The message will be handled once more."

#: src/vigilo/correlator/actors/rule_runner.py:42
#, python-format
msgid ""
"Got an exception while running rule \"%(rule)s\". Running the correlator "
"in the foreground (service vigilo-correlator debug) may help "
"troubleshooting (%(error)s)"
msgstr ""
"Got an exception while running rule \"%(rule)s\". Running the correlator "
"in the foreground (service vigilo-correlator debug) may help "
"troubleshooting (%(error)s)"

#: src/vigilo/correlator/rules/example.py:37
#, python-format
msgid "id %(id)s payload %(payload)s"
msgstr "id %(id)s payload %(payload)s"

#: src/vigilo/correlator/rules/svc_on_host_down.py:48
msgid "Host is down"
msgstr "Host is down"

#: src/vigilo/correlator/rules/svc_on_host_down.py:50
#, python-format
msgid "Setting %d services to UNKNOWN"
msgstr "Setting %d services to UNKNOWN"

#: src/vigilo/correlator/rules/svc_on_host_down.py:117
#, python-format
msgid "Unsupported transition: %(from)s -> %(to)s"
msgstr "Unsupported transition: %(from)s -> %(to)s"

#: src/vigilo/correlator/rules/svc_on_host_down.py:126
#, python-format
msgid "Asking Nagios for updates on %d services"
msgstr "Asking Nagios for updates on %d services"

